index.js

```diff
@@ -1,37 +1,148 @@
+}
+
+
+// Powered by Mapquest Geocode API
+// https://developer.mapquest.com/documentation/geocoding-api/address/get/
+
+async function getDecimalLocation(Location) {
+    const apikey = process.env.MapQuest_API_KEY
+
+    const response = await axios.get(`http://www.mapquestapi.com/geocoding/v1/address?key=${apikey}&location=${Location}&maxResults=1`)
+    const lat = response.data.results[0].locations[0].latLng.lat
+    const lng = response.data.results[0].locations[0].latLng.lng
+    return `${lat},${lng}`
+}
+
+// https://www.epa.gov/sunsafety/uv-index-scale-0
+// UV safety information sourced from United States EPA website
+
+function getRiskText(uvIndex) {
+    switch (uvIndex) {
+        case 0:
+        case 1:
+        case 2:
+            return 'Low danger from UV levels for the average person. No sunscreen required'
+        case 3:
+        case 4:
+        case 5:
+            return 'Moderate risk of harm from current UV levels. Be sure to apply sunscreen especially if you plan on being outside around midday.'
+        case 6:
+        case 7:
+            return 'High risk of harm from current UV levels. Sunscreen should be applied to prevent burns. If possible, avoid sun exposure between ten oclock and four oclock.'
+        case 8:
+        case 9:
+        case 10:
+            return 'Very high risk of harm from UV exposure. Burns can occur quickly unless sunscreen is applied every twp hours and unprotected sun is avoided between ten oclock and four oclock.'
+        case 11:
+            return 'Extreme risk of burns from UV exposure. Unprotected skin and eyes can burn in minutes.'
+        default:
+            return 'Please try again.'
+    }
 }
 
 const sunscreenIntentHandler = {
     canHandle(handlerInput) {
-        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
-            && handlerInputInput.requestEnvelope.request.intent.name === 'sunscreenIntent'
+        const {
+            request
+        } = handlerInput.requestEnvelope
+
+        return request.type === 'IntentRequest' && request.intent.name === 'sunscreenIntent'
     },
-    handle(handlerInput) {
-        const uvDate = handler.Input.requestEnvelope.reuest.intent.slots.Date.value
-        const uvLocation = handler.Input.requestEnvelope.reuest.intent.slots.Location.value
+    async handle(handlerInput) {
+        const uvLocation = await getDecimalLocation(handlerInput.requestEnvelope.request.intent.slots.Location.value)
         const reprompt = 'Would you like more information?'
 
-        return getUvIndex(uvDate, uvLocation).then((uvIndex) => {
-            const speechText = `The current UV Index is ${uvIndex}`
+        const uvIndex = await getUvIndex(uvLocation)
+        const speechText = `The current UV Index is ${uvIndex}. `
+        const riskText = getRiskText(uvIndex)
+
+        return handlerInput.responseBuilder
+            .speak(speechText + riskText)
+            .reprompt(reprompt)
```

Comment:
The user is going to say Yes or No to this and you'll throw the FallbackIntent instead of handling it.